elastic/synthetics
# Allow users to bundle node_modules

> Please notice that I had to rebase this PR on top of #576 so that we can actually parse streaming responses from Kibana.


# Summary

This PR allows users to bundle modules into the scripts they push.

I've marked it as draft as it still needs E2E tests.


# How to test this PR

To test this PR, try including a JS library into your journey, by importing it into the journey file.

Then push your journey with the extra imported module to Kibana.

You should be able to run the Journey just fine.
